src/js/extensions/paste.js

```diff
@@ -176,11 +176,6 @@ var PasteHandler;
                     workEl.innerHTML = workEl.innerHTML.replace(/\n/gi, ' ');
 
                     switch (workEl.tagName.toLowerCase()) {
-                        case 'a':
```

Comment:
Well I've tested on Chrome and Firefox (on my Mac) and `_target` attribute was never created on links, event with the option enabled. When I `console.log` the element (`workEl`) it seems to properly have the attribute, but the dom doesn't have it ..

I then checked the non-multiline support (in the `else`) and see that targetBlank was never called, and so never apply on pasting non-multiline content. That's why I moved the targetBlank `if` in `pasteHTML`. And this function is called on multiline support (line 162).

This way:
- I fixed the element which wasn't properly updated with `_target` attribute on multiline pasting
- I added targetBlank support on non-multiline pasting